.github/workflows/validate-migration-scripts/compile-openssl.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+SCRIPT_ROOT="$(dirname "$(realpath "${BASH_SOURCE[0]}")")"
+cd "${SCRIPT_ROOT}/../../.."
+
+set -euxo pipefail
+
+pushd "${OPENSSL_SOURCE_DIR}"
+
+CONFIG_ARGS=(
+  --prefix="${OPENSSL_INSTALL_DIR}"
+)
+
+read -r -a array <<<"${OPENSSL_FLAGS:-}"
+for flag in "${array[@]}"; do
+  if [[ -n "${flag}" ]]; then
+    CONFIG_ARGS+=("$flag")
+  fi
+done
+
+./config "${CONFIG_ARGS[@]}"
+make "-j$(nproc)"
```

Comment:
nit: Maybe `make -j "$(nproc)"` instead? It seems a little weird to see the `-j` inside the string. It's also not obvious that this syntax is supported based on `make --help` on Linux:

```
-j [N], --jobs[=N]          Allow N jobs at once; infinite jobs with no arg.
```